CoEpi/domain/model/Alert.swift

```diff
@@ -22,7 +21,15 @@ struct Alert: AutoEquatable {
     let other: Bool
     let noSymptoms: Bool
 
+    var isRead: Bool
+
     var durationSeconds: Int {
         Int(end.value - start.value)
     }
+
+    func changing(change: (inout Alert) -> Void) -> Alert {
```

Comment:
@i-schuetz This does not seem to be used.

Reply:
https://github.com/Co-Epi/app-ios/pull/329